### DISCUSSION TITLE
Get `tags` on any other page than `DocTagDocListPage`

### DISCUSSION THREAD

**Discussion by niklasp** [facebook/docusaurus#8931] [Q&A]
I want to display tags alongside there count on the index page of my website. The `DocTagDocListPage` receives a `tags` attribute that looks like

```
[
  {
    "label": "contribute",
    "permalink": "/docs/tags/contribute",
    "count": 1
  },
  {
    "label": "authenticaation",
    "permalink": "/docs/tags/authentication",
    "count": 1
  }
]
```

How can I get that? Do I need to write my own theme or plugin for that? 

 I alredy asked in discord and was directed here. I appreciate your help.

The overall idea is to have a component on the home page that lists e.g. 2 latest pages for each tag with a page title and other meta information.

**Comment by slorber** [comment]
Unfortunately Docusaurus is modular, plugins don't see each other, and we don't have a good way to let one plugin (blog) pass data to a page created by another plugin (page plugin - homepage).

There are multiple workarounds (extending the blog plugin, having a pre-build step to generate a json file with those data...) but none are really great and you'd better hardcode that data for now.

The good news is that I believe Server Components will help us achieve this flexibility so that you can cherry-pick any plugin-exposed data anywhere in the React tree (ie in your homepage too). But we are not there yet.

---

Not exactly the same use-case, but a little bit related to https://github.com/facebook/docusaurus/issues/8886
